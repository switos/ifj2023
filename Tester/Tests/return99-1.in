func generateCrash() {
    let a = 1
    let b = 0
    let result = a / b
    print("Result: \(result)")
}

generateCrash()
