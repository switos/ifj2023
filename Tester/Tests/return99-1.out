Expected Output:

Expected Return Code: 99
